lib/sla.js

```diff
@@ -84,12 +123,38 @@ function createChannel(client, bridge) {
  * @param {Object} client - Client received from app.js.
  * @return {Q} Q promise object.
  */
-module.exports = function(client) {
-  return designateMixingBridge(client)
-    .then(function (bridge) {
-      return createChannel(client, bridge)
-        .then(function (channel) {
-          return addChannelsToBridge(channel, bridge);
-        });
-    });
+module.exports = function(client, channel, bridgeNumber) {
```

Comment:
You should rename channel to inbound to keep consistent with your naming scheme.